modules/apps/frontend-js/frontend-js-react-web/package.json

```diff
@@ -1,14 +1,5 @@
 {
 	"dependencies": {
-		"@clayui/icon": "3.40.0",
-		"@liferay/frontend-js-state-web": "*",
-		"classnames": "2.3.1",
-		"formik": "2.1.4",
-		"prop-types": "15.7.2",
-		"react": "16.12.0",
-		"react-dnd": "11.1.1",
-		"react-dnd-html5-backend": "11.1.1",
-		"react-dom": "16.12.0"
```

Comment:
@ling-alan-huang, `frontend-js-react-web` is a module that should be handled as an exception since it's listed [here](https://github.com/liferay/liferay-portal/blob/32c45c094da579177df1724e45846a07c56fc86b/modules/npmscripts.config.js#L44-L53), but, apparently, it was handled as a common module. Can you please check it?

Reply:
Hi @ling-alan-huang, I was analyzing the commits and both related failures (`frontend-js-a11y-web` and `frontend-js-react-web`) apparently are related to this [commit](https://github.com/natocesarrego/liferay-portal/pull/1177/commits/a23a6f0c8de61b35ae325fab2066da68cb446d7a), as you pointed above. :slightly_smiling_face:

Should we remove the following statement?
```
if (!dependencyName.startsWith("@")) {
    excludedDirNames.add(dependencyName);
}
```

Reply:
@natocesarrego yes, we can remove it, but we still need a logic to remove `@liferay/` from `@liferay/frontend***`, and put into `excludedDirNames` list.

Reply:
I see, @ling-alan-huang. I think I'll give this pull back to you because when I run SF locally, the new rule apparently isn't applied. Can you please work on the logic mentioned by you above?

Thanks

Reply:
I will resend to you today.